.github/workflows/integration-test-helm-suite.yaml

```diff
@@ -73,8 +73,9 @@ jobs:
         path: /home/runner/work/rook/rook/tests/integration/_output/tests/
 
     - name: setup tmate session for debugging
-      if: failure() && contains(github.event.pull_request.labels.*.name, 'debug-helm-suite')
+      if: failure()
       uses: mxschmitt/action-tmate@v3
+      timeout-minutes: 120
```

Comment:
Didn't we change these timeouts to 60?

Reply:
we changed in case of push event. right?

Reply:
Ok, what if we change these as well? That could be a separate PR though since it's not in master yet.

Reply:
yes, we can change but IIRC we decided to change for push event and in the canary test only, right?

Reply:
Is there a need for a longer timeout for the integration tests? Seems like 60 min is sufficient.

Reply:
created pr in master #8557